.eslintrc

```diff
@@ -18,8 +18,13 @@
       "files": [ "**/*.ts" ],
       "extends": [
         "eslint:recommended",
-        "prettier"
-      ]
+        "prettier",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"    
+      ],
+      "rules": {
+        "@typescript-eslint/no-inferrable-types": "off"
+      }
```

Comment:
should not it have a trailing comma?

Reply:
This file was not with trailing commas. Look at your comment on `prettier`.